src/screens/assetDetails/screen/assetDetailsScreen.tsx

```diff
@@ -104,6 +104,25 @@ const AssetDetailsScreen = ({ navigation, route }: AssetDetailsScreenProps) => {
     }
   };
 
+  const _handleCoinRepeat = (coinActivity: CoinActivity) => {
+    let parsedUsername = '';
+
+    // from @user1 to @user2
+    parsedUsername = coinActivity.details?.split(' ').pop() || '';
```

Comment:
perhaps instead of post processing details string we can extend activity type to support to and from properties, as details string can be subject to change. what do you think...?

Reply:
@noumantahir  correct me If I am wrong, but we still have to use detail string to populate to and from? Also not all tokes support to and from..

Reply:
I have added support for sender and receiver, but I think it only works for points.

Reply:
@gamingumar essentially the object we groom/build out of various different types of operations is custom made and can be modified as per requirement... if you check `groomingTransactionData` in /wallet.ts, this is the method we use to make string that eventually we are processing to get preferred username, which can be inconsistent or unnecessary from we can easily get to and from properties out opData of transfer, similar approach should be useable for engine data as well... let me know what you think....

Reply:
@noumantahir can you check now